### ISSUE TITLE
Interactions avec Javascript should use `globalThis` instead of `window`

### ISSUE THREAD

**Issue by denis-migdal** [brython-dev/brython#2524]
In the documentation : https://brython.info/static_doc/3.13/fr/jsobjects.html

Examples should use `globalThis` instead of `window` :
```python
from browser import globalThis
```

Indeed, `globalThis` is the standard way to access global values, working in window and non-window context, cf https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

Brython does remove `browser.window` in Node and WebWorker contexts.

**Comment by denis-migdal** [comment]
Note: `browser.globalThis` doesn't seem to be supported by Brython, but `browser.self` is.

I think self is standard in window environment and Webworker, but might not be in other environment (even if they seem to implement it).

So at least, examples should use `from browser import self` to access JS global variables.

**Comment by denis-migdal** [comment]
Also : `self` would be confusing inside methods where we use `self` like the JS `this`.

**Comment by PierreQuentel** [comment]
I agree with the idea of providing a universal name for all contexts, but `globalThis` is so ugly... Too long, and explicitely refering to the Javascript language whereas the object is a collection of names for Javascript _and_ the Web API.

In the commit above I introduce the name `scope`. I would have prefered `global` but it's already a Python keyword... Is it ok ?

**Comment by denis-migdal** [comment]
I think "scope" alone may be confusing.

It seems this corresponds to a "global scope" (cf https://developer.mozilla.org/en-US/docs/Glossary/Global_scope ), but "globalScope" might be too long.

A "gscope" or "gs" would not be explicit enough i think.
Ofc, it'd be always possible to write :
```python
from browser import globalScope
gs = globalScope
```
But this is kind of not practical.

So "scope" isn't ideal, but I do not really have a better idea.
